### PR TITLE
Coding schemes for this project

### PR DESCRIPTION
**Pull request by princelySid**

### CODE REVIEW

File: coding_schemes/Cash_Modality_Yesno.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "Prefer current modality",
+      "NumericValue": 0,
+      "StringValue": "prefer_current_modality",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-5f7c2445b240",
+      "CodeType": "Normal",
+      "DisplayText": "Safety ",
```
Comment (as2388) [inline coding_schemes/Cash_Modality_Yesno.json]: any reason for the space here?

File: coding_schemes/Cash_Modality_Yesno.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "Inclusive access",
+      "NumericValue": 4,
+      "StringValue": "inclusive_access",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-439a68abb1bf",
+      "CodeType": "Normal",
+      "DisplayText": "Prevents corruption ",
```
Comment (as2388) [inline coding_schemes/Cash_Modality_Yesno.json]: and here

File: coding_schemes/Cash_Modality_Yesno.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+      "NumericValue": 5,
+      "StringValue": "prevents_corruption_",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-2425e532273a",
+      "CodeType": "Normal",
+      "DisplayText": "Long queues ",
```
Comment (as2388) [inline coding_schemes/Cash_Modality_Yesno.json]: and here etc

File: coding_schemes/Inclusion_Yes_No.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
+{
```
Comment (as2388) [inline coding_schemes/Inclusion_Yes_No.json]: This file is called yes_no but doesn't contain the codes yes/no?

File: coding_schemes/demog_schemes.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Clan
```
Comment (as2388) [inline coding_schemes/demog_schemes.csv]: Not sure if this should be committed

File: coding_schemes/Cash_Modality_Yesno.json (hunk 1 above)
Comment (princelySid) [reply coding_schemes/Cash_Modality_Yesno.json]: This is because of csv, they're generated from.

**Comment by as2388** [comment]
> This is because of csv, they're generated from.

Ok, then why are there spaces in the csv?

File: coding_schemes/Inclusion_Yes_No.json (hunk 4 above)
Comment (princelySid) [reply coding_schemes/Inclusion_Yes_No.json]: It's what the variable is called in rapidpro, there's a separate file for yes_no

File: coding_schemes/demog_schemes.csv (hunk 5 above)
Comment (princelySid) [reply coding_schemes/demog_schemes.csv]: Why?

**Comment by princelySid** [comment]
> This is because of csv, they're generated from.
> Ok, then why are there spaces in the csv?

Oversight on my part

File: coding_schemes/demog_schemes.csv (hunk 5 above)
Comment (lukechurch) [reply coding_schemes/demog_schemes.csv]: We don't commit the CSVs - they're not the canonical form of the coding schemes - the canonical form is the JSON files. Please could you remove this.

File: coding_schemes/pdm_schemes.csv [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Needs_Met_Yesno,Cash_Modality_Yesno,Community_Priorities,Inclusion_Yes_No,Yes_No
```
Comment (lukechurch) [inline coding_schemes/pdm_schemes.csv]: As above please

Comment (princelySid) [reply coding_schemes/pdm_schemes.csv]: Removed the csvs

File: coding_schemes/Inclusion_Yes_No.json (hunk 4 above)
Comment (as2388) [reply coding_schemes/Inclusion_Yes_No.json]: Is there a more appropriate term we can use that matches what RDA would call this?
For projects I've worked on, something like 'Inclusion_Reason' would usually be appropriate, but I'm not sure what terminology RDA are using on this one

File: coding_schemes/Age.json [hunk 7]
```diff
@@ -0,0 +1,1060 @@
...
+      "ControlCode": "NIC",
+      "DisplayText": "NIC (not internally consistent)",
+      "NumericValue": -50,
+      "StringValue": "NIC",
+      "VisibleInCoda": false
+    },
+    {
+      "CodeID": "code-STOP-08b832a8",
+      "CodeType": "Control",
+      "ControlCode": "STOP",
```
[the remark this answers is not in the document]
Comment (princelySid) [reply coding_schemes/Age.json]: These are added automatically by the creation script

File: coding_schemes/Clan.json [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "Ajuuran",
+      "NumericValue": 2,
+      "StringValue": "ajuuran",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-160755ca2f2c",
+      "CodeType": "Normal",
+      "DisplayText": "Marehan /Mareexan",
```
[the remark this answers is not in the document]
Comment (princelySid) [reply coding_schemes/Clan.json]: Where?

**Comment by princelySid** [comment]
https://github.com/AfricasVoices/Project-MAAP-Somalia/pull/11#discussion_r246340732

File: coding_schemes/Clan.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (as2388) [reply coding_schemes/Clan.json]: before the /

[the remark this answers is not in the document]
Comment (princelySid) [reply coding_schemes/Clan.json]: Fixed it.

**Comment by princelySid** [comment]
https://github.com/AfricasVoices/Project-MAAP-Somalia/pull/11#discussion_r246340732
So far I'm just using what RDA calls it.

**Pull request merged**